tests/end_to_end/test_tables.py

```diff
@@ -0,0 +1,99 @@
+import pytest  # noqa: F401
+
+import logging
+from pathlib import Path
+import sys
+
+import yaml
+
+from n2y.main import cli_main
+
+
+def test_simple_table(caplog, monkeypatch, request, tmp_path, valid_access_token):
+    """
+    Simply show what flattened Markdown content is expected if the input
+    contains tables without headers.
+
+    Relies on https://www.notion.so/Simple-Tables-9b1dd705f61647b6a10032ec7671402f?pvs=4
+    """
+    object_id = "9b1dd705f61647b6a10032ec7671402f"
+
+    config = {
+        "exports": [
+            {
+                "id": object_id,
+                "node_type": "page",
+                "output": f"{request.node.name}-output",
+                "pandoc_format": "gfm",
+            }
+        ]
+    }
+    with monkeypatch.context() as m:
```

Comment:
I think there are a few lessons that've made pytest more opinionated about this. People trying to run concurrently (test or plugin) will hit thread problems when their process calls an os.chdir or sets an environment variable, so the advice is to monkeypatch and/or use absolute paths. There are other examples, too, and I'm not debating their relevance but whether they come officially from within that project.

Reply:
I can see how calling `os.chdir` wouldn't work in a multi-threaded environment. I don't think the approach in https://github.com/innolitics/n2y/blob/main/tests/test_end_to_end.py#L18-L46 uses environment variables.

If this is easy to fix and the solution is simple and doesn't make the code much more complicated, I suggest fixing it here:

https://github.com/innolitics/n2y/blob/main/tests/test_end_to_end.py#L18-L46

and making your test follow the approach of the existing tests.

If it is not easy to fix, please create an issue in the task backlog and explain the problem and a proposed solution and approximately how much time you think it will take to fix. We can then decide if it's important to fix during our next sprint planning call.

Overall, the goals are to (1) focus on keeping the code simple and following existing patterns that are already set, (2) meet the requirements of the specified tasks in a timely manner.

Reply:
It probably doesn't rise to be an Issue, and my experience is that fitting pytest fixtures into helper functions does add boilerplate -- would be counterproductive to goal (1). To meet goal (2) I just refactored this into the test_end_to_end module. Should be much shorter amount to review.